website/docs/concepts/function-components/pure-components.mdx

```diff
@@ -32,5 +29,5 @@ fn HelloWorld(props: &Props) -> Html {
 
 ## Un-pure components
```

Comment:
*un*pure or *im*pure?

Reply:
*Impure* seems to be what react is using, but both words are ... words. Changed to impure though.